Mathlib/Data/MvPolynomial/Basic.lean

```diff
@@ -255,6 +255,12 @@ theorem C_inj {σ : Type*} (R : Type*) [CommSemiring R] (r s : R) :
   (C_injective σ R).eq_iff
 #align mv_polynomial.C_inj MvPolynomial.C_inj
 
+instance nontrivial_of_nontrivial (σ : Type*) (R : Type*) [CommSemiring R] [Nontrivial R] :
```

Comment:
Do we have this instance for `MonoidAlgebra`? If so it should be inheritable with a one-liner.